lib/client-assets.php

```diff
@@ -1193,6 +1019,7 @@ function gutenberg_editor_scripts_and_styles( $hook ) {
 			'post' => $post->ID,
 		)
 	);
+	wp_tinymce_inline_scripts();
```

Comment:
See: https://github.com/WordPress/wordpress-develop/blob/6edf51566187f1bba6aa5d891a681b02996ccef4/src/wp-admin/edit-form-blocks.php#L334

Reply:
@iseulde Do we need to submit a patch upstream to remove this line given recent refactorings?